src/Components/Home/Deals.js

```diff
@@ -0,0 +1,26 @@
+import DealsProducts from "./DealsProducts"
+
+function Deals() {
+    return (
+        <div className="card-container">
+                
+                    <DealsProducts 
```

Comment:
you can you use map method to map over here

Reply:
Could you please elaborate a little as of how to use map over here. As of now I'm trying to pass props to the components.

Reply:
you can take image,itemName,discount in array and use map over here

Reply:
sure got it